etl-pipeline/tests/unit/api/blueprints/test_api_collection_blueprint.py

```diff
@@ -19,1069 +18,813 @@
-
-            mock_feed_construct.assert_called_once_with(collection, mock_db)
-
-            mock_utils["formatOPDS2Object"].assert_called_once_with(
-                200, "testOPDS2Feed"
-            )
-
-    def test_collection_update_error(self, test_app, mock_utils, mocker):
-        mock_db = mocker.MagicMock(session=mocker.MagicMock())
-        mock_db_client = mocker.patch("api.blueprints.drbCollection.DBClient")
-        mock_db_client.return_value = mock_db
-
-        mock_db.fetchUser.return_value = mocker.MagicMock(
-            user="testUser", password="testPswd", salt="testSalt"
-        )
-
-        mock_utils["formatResponseObject"].return_value = "testErrorResponse"
-
-        mock_db.fetchSingleCollection.return_value = mocker.MagicMock(uuid="testUUID")
-
-        mocker.patch.dict(os.environ, {"NYPL_API_CLIENT_PUBLIC_KEY": "test"})
-
-        mock_base_64 = mocker.patch("api.blueprints.drbCollection.b64decode")
-        mock_base_64.return_value = b"testUser:testPswd"
-
-        with test_app.test_request_context(
-            "/update/testUUID", headers={"Authorization": "Basic testAuth"}
-        ):
-            test_api_response = collectionUpdate("testUUID")
-
-            assert test_api_response == "testErrorResponse"
-
-            mock_utils["formatResponseObject"].assert_called_once_with(
-                400,
-                "updateCollection",
-                {
-                    "message": "At least one of these fields(title, creator, description, etc.) are required"
-                },
-            )
-
-    def test_static_collection_createsuccess(self, test_app, mock_utils, mocker):
-        mock_db = mocker.MagicMock(session=mocker.MagicMock())
-        mock_db_client = mocker.patch("api.blueprints.drbCollection.DBClient")
-        mock_db_client.return_value = mock_db
-
-        mock_db.fetchUser.return_value = mocker.MagicMock(
-            user="testUser", password="testPswd", salt="testSalt"
-        )
-
-        collection = mocker.MagicMock(uuid="testUUID")
-        mock_db.createStaticCollection.return_value = collection
-
-        mock_feed_construct = mocker.patch(
-            "api.blueprints.drbCollection.constructOPDSFeed"
-        )
-        mock_feed_construct.return_value = "testOPDS2Feed"
-
-        test_request_body = {
+@pytest.fixture
+def mock_utils(mocker):
+    return mocker.patch.multiple(
+        APIUtils,
+        formatResponseObject=mocker.DEFAULT,
+        formatOPDS2Object=mocker.DEFAULT,
+        validatePassword=mocker.DEFAULT,
+    )
+
+
+@pytest.fixture
+def mock_db_and_client(mocker):
+    mock_db = mocker.MagicMock(session=mocker.MagicMock())
+    mock_db.fetchUser.return_value = mocker.MagicMock(
+        user="testUser", password="testPswd", salt="testSalt"
+    )
+    mock_db_client = mocker.patch("api.blueprints.drbCollection.DBClient")
+    mock_db_client.return_value = mock_db
+    return mock_db, mock_db_client
+
+
+# @pytest.fixture
```

Comment:
Do we want to keep this fixture commented? If it's no longer needed, let's remove these lines!